modules/floating-buttons/assets/scss/floating-bars/widgets/floating-bars.scss

```diff
@@ -259,16 +259,12 @@
 	}
 }
 
-.e-con:has(.e-floating-bars) > .e-con-inner {
-	padding-block-end: 0;
-	padding-block-start: 0;
-}
-
-.elementor-element:has(.e-floating-bars:is(:not(.is-sticky))) {
+.e-con.e-con--floating-bars {
```

Comment:
I would keep the :has() reference.
And use the .e-con-inner--floating-bars as fallback.
So that modern browsers will load the correct styling immediately, even before the JS functions are running.